Publish initial draft of the emnify Portal user guide (#142)
emnify/product-docs: docs/portal/organization/sso.md

---
description: Improve network security and simplify user administration
slug: /sso
---

# Single sign-on

The emnify Portal offers single sign-on (SSO) with select providers.

> SSO provides several benefits, including improved network security and simplified administration.

Reviewer: ```suggestion
SSO improves account security and simplifies user administration.
```

Network security can be understood as their cellular network, the network of devices -- so a bit unclear. If we can't list several additional benefits, we can be specific here to the two we are certain about.

:::info

> Single sign-on (SSO) is only available with the [**Pro** plan](https://portal.emnify.com/organisation-settings/subscription).

Reviewer: ```suggestion
Single sign-on (SSO) comes with the [**Pro** plan](https://portal.emnify.com/organisation-settings/subscription).
```
